tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/MsTestIntegration.cs

```diff
@@ -80,6 +80,7 @@ internal static class MsTestIntegration
         ["b754cc51-34ed-419c-8582-bff04c3db05f"] = "3.8.0",
         ["2b3d62e3-5607-4ebd-840e-ee80475cc0bc"] = "3.8.1",
         ["3fe23123-93a2-4c44-8219-0a5f27a10316"] = "3.8.2",
+        ["e7bba6ac-32f6-4e62-9a3a-cf259c9e7448"] = "3.9.1",
```

Comment:
Shouldn't we have `3.9.0` too?

Reply:
done in [0db15ca](https://github.com/DataDog/dd-trace-dotnet/pull/7070/commits/0db15ca09c2ba13fb50db81c6ca7d527592886df)